applications/StructuralMechanicsApplication/custom_elements/base_shell_element.cpp

```diff
@@ -439,9 +487,10 @@ void BaseShellElement::CheckProperties(const ProcessInfo& rCurrentProcessInfo)
     {
         CheckSpecificProperties();
 
+        // TODO is this needed???? => it is, the dummy is needed for "Check" => unify!
         ShellCrossSection::Pointer dummySection = ShellCrossSection::Pointer(new ShellCrossSection());
         dummySection->BeginStack();
-        dummySection->AddPly(props[THICKNESS], 0.0, 5, pGetProperties());
```

Comment:
one question: what happens if the thickness changes? for a large deformation shell that would definitely be a problem (although our shells as of now are large rotation but not large deformation)

Reply:
currently: nothing :)